script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -788,12 +832,13 @@ The driver hasn't said a single word.
 [sha:4b2fdeb90b55fd854bab718115d467fc228a7b69]{
 -- Page 937, Offset 26986.
 -- 遠野志貴は後部座席に、シートベルトで固定されている。
-I, Tohno Shiki, am strapped to the back seat with a seatbelt.
+Tohno Shiki is strapped to the back seat with a seatbelt.
 }
 [sha:4bac6e57558ff8406a4ca742a38bf933c33729ce]{
 -- Page 937, Offset 26987.
 -- 外の空気も、中の空気も重苦しい。
-The air outside and the air inside feel heavy.
+// Supposed to be an 空気 (gaseous) and 空気 (mood) juxtapos? Or same/same we think?
```

Comment:
definitely, but this is one of those cases where I think it's ok to just say fuck it